frontend/src/App.js

```diff
@@ -1,16 +1,46 @@
 import React from "react";
 import "./styles/App.css";
-import { BrowserRouter, Switch, Route } from "react-router-dom";
 
+import {BrowserRouter, Route, Switch} from "react-router-dom";
 // Components
 import Navigation from "./components/navigation";
 
+// import { BrowserRouter, Switch, Route } from "react-router-dom";
+// import { Link } from 'react-router-dom';
+// import QuantumSimulator from "./components/quantum";
+// import Header from "./components/header";
+// import Footer from "./components/footer";
+
+
+// // Components
+// import { makeStyles } from '@material-ui/core/styles';
+// import AppBar from '@material-ui/core/AppBar';
+// import Toolbar from '@material-ui/core/Toolbar';
+// import Typography from '@material-ui/core/Typography';
+// import Button from '@material-ui/core/Button';
+
```

Comment:
If this code isnt doing anything can we remove it and merge it in later when its ready?

Reply:
deleted

Reply:
Deleted this while resolving conflict.